packages/itwinui-react/src/core/Select/Select.tsx

```diff
@@ -70,6 +70,8 @@ export type SelectOption<T> = {
   size?: 'default' | 'large';
   /**
    * Value of the item.
+   *
+   * Pass `null` to reset the value.
```

Comment:
Should this line be for the `value` in `SelectMultipleTypeProps` instead?

Reply:
> though, strangely, i cannot get the jsdocs to show up

Ohh, same for me, maybe something we can look into a future PR. If so, maybe we can add a TODO for this in the PR description.

Reply:
Done